PackageExplorer/PackageChooser/PackageChooserDialog.xaml.cs

```diff
@@ -220,5 +210,27 @@ private void PackageSourceBox_OnSelectionChanged(object sender, SelectionChanged
 
             e.Handled = true;
         }
+
+        private void ListBoxPackages_ScrollChanged(object sender, ScrollChangedEventArgs e)
+        {
+            if (e.VerticalChange != 0 && e.OriginalSource is ScrollViewer scrollViewer && scrollViewer.VerticalOffset == scrollViewer.ScrollableHeight)
+            {
+                if (scrollViewer.VerticalOffset == 0 && scrollViewer.ScrollableHeight == 0) // scroll back to top if packages got cleared
+                {
+                    scrollViewer.ScrollToTop();
+                }
+                else if (_viewModel.IsEditable && _viewModel.HasMore)
```

Comment:
Maybe move this check to the LoadedCommand CanExecute action.
And rename it to `LoadMoreCommand`.

Reply:
👍